docs/snapshotting/snapshot-support.md

```diff
@@ -38,7 +36,23 @@ This has the advantage of very fast snapshot loading times, but comes with the c
 of having to keep the guest memory file around for the entire lifetime of the
 resumed microVM.
 
-*Note*: Snapshotting is currently supported only on `x86_64` machines.
+*Note*: Snapshotting is currently supported on Intel and AMD Zen `x86_64` CPUs.
+
+## Functionality and performance
+The current Firecracker snapshot tests and support are targeting only Intel Skylake, Cascade Lake and AMD Zen2 CPUs. These are the officially supported CPUs, but the functionality *may* work on others as well. Our plan is to incrementally extend support for other platforms and CPUs.
```

Comment:
I would move this above and rephrase "Snapshotting is supported on Intel Skylake, Cascade Lake and AMD Zen2 CPUs, but the functionality *may* work on others Intel and AMD x86_64 CPUs with virtualization support."

Reply:
makes sense.

Reply:
done